Makefile

```diff
@@ -3,6 +3,10 @@ default: test
 lint:
 	golangci-lint run ./...
 
+tools:
+	go install google.golang.org/protobuf/cmd/protoc-gen-go
```

Comment:
It's a bummer this cannot also be moved (e.g. imports are baked into the `.pb.go` generated files) 😄

Reply:
Agreed 😞